salt/master.py

```diff
@@ -2030,12 +2030,35 @@ def get_token(self, clear_load):
             return False
         return self.loadauth.get_tok(clear_load['token'])
 
+    def publish_batch(self, batch, clear_load):
+        batch_load = {}
+        batch_load.update(clear_load)
+        batch_load['kwargs']['batch'] = batch
+        import salt.cli.batch_async
```

Comment:
Is it intended to have import here and not at the beginning of the file?

Reply:
:+1:

Reply:
keeping it